### ISSUE TITLE
Add delete post functionality

### ISSUE THREAD

**Issue by krishnaacharyaa** [krishnaacharyaa/wanderlust#40]
Once #38 #37 is resolved, this can be taken up, this should add in the delete post functionality calling the respective api from backend which already exists, and confimation dialog also needs to be prompted when the user wishes to delete the post
And it should also include the test cases for the developed functionality

**Comment by SanjeevKrSingh** [comment]
Hey @krishnaacharyaa  Is anyone doing this? If not could you please assign this to me.

**Comment by krishnaacharyaa** [comment]
Post #39, we'll take this up

**Comment by chaitya-titan** [comment]
Hey @krishnaacharyaa ... could you assign this issue to me

**Comment by krishnaacharyaa** [comment]
how are you planning to work on this? @chaitya-titan

**Comment by chaitya-titan** [comment]
As far as I can see, the delete route is already created in the backend... so after the user auth is done... will add a delete icon on the post(if the post is created by the user)... give a confirmation dialog box on the screen to 'confirm delete'... if there is a delete icon present with you... will probably use that... on confirm delete... make an API call to "deletePostByIdHandler" route... 

This is the flow I am thinking as of now...

**Comment by krishnaacharyaa** [comment]
Yeah, but we would need to focus on auth first to get it in place. Before that we are focusing on adding test cases milestone. Once these are done we'll come to this and assign...

**Comment by chaitya-titan** [comment]
Ok Thank you

**Comment by Shivam2534** [comment]
Hello Sir,

My name is Shivam Kanchole, and I am a fourth-year student at IIT Roorkee. I find this issue both interesting and challenging. Additionally, I have worked on similar projects in the past. Could you please assign this issue to me?

Thank you.

Best regards,
Shivam Kanchole
